common/cluster/metadata.go

```diff
@@ -104,6 +104,8 @@ type (
 		InitialFailoverVersion int64 `yaml:"initialFailoverVersion"`
 		// Address indicate the remote service address(Host:Port). Host can be DNS name.
 		RPCAddress string `yaml:"rpcAddress"`
+		// Cluster ID allows to explicitly the ID of the cluster. Optional.
+		ClusterID  string `yaml:"clusterId"`
```

Comment:
maybe this should be ignored too if set in .yaml files?

Reply:
I think that'd be great, I don't see a reason for anyone to set this but our CLI.